src/app/auth/signup.tsx

```diff
@@ -44,18 +87,78 @@ function SignUpScreen() {
         <Input
           label="Password"
           leftIcon={{ type: 'font-awesome', name: 'lock' }}
-          onChangeText={text => setPassword(text)}
+          onChangeText={text => checkPassword(text)}
           value={password}
           secureTextEntry
           placeholder="Password"
           autoCapitalize="none"
         />
+      </View>
+      <View>
+        {password !== '' && (
+          <View style={globalStyles.passwordComplexity}>
```

Comment:
We don't want to add the styles for passwordComplexity to globalStyles, because this style is only going to be used on this screen. Create a styles sheet at the bottom of this file (look at verify.tsx as an example) and move this set of styles there instead.

Reply:
Also looking at the designs, we have 8 pixels of padding between each password complexity line of text. If you add paddingBottom: 8 to your passwordComplexity styles, this should fix it.